src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -190,23 +190,19 @@ def _initialize_scale_zero_point(
         register_offload_parameter(module, f"{base_name}_g_idx", init_g_idx)
 
 
-def _initialize_attn_scales(module: Module) -> None:
-    """Initlaize k_scale, v_scale for  self_attn"""
+def _initialize_attn_scales(module: Module):
+    """Initlaize k_scale, v_scale for self_attn"""
 
     expected_shape = 1  # per tensor
 
-    param = next(module.parameters())
```

Comment:
WHAAAAAAAAAAAAAAT 😨 we just assumed next(model.parameters()) give us the weight param

Reply:
Sorry this needs to be reworked. This is meant to be called on an "attention" module, so we need to get the k_proj and v_proj attributes, not the weight attribute.

But yes, making any assumption of ordering on `module.parameters()` is incorrect